packages/web3/src/contract/ralph.ts

```diff
@@ -503,7 +521,7 @@ function _encodeField<T>(fieldName: string, encodeFunc: () => T): T {
     return encodeFunc()
   } catch (error) {
     if (error instanceof Error) {
-      throw new Error(`Invalid ${fieldName}, error: ${error.message}`)
+      throw new Error(`Invalid ${fieldName}, error: ${error.message} `)
```

Comment:
Why do we add an empty space?

Reply:
Caused by the editor config, fixed, thanks.